src/dpapi_ng/_client.py

```diff
@@ -226,6 +230,65 @@ def _decrypt_blob(
     )
 
 
+def _encrypt_blob(
+    blob: bytes,
+    key: GroupKeyEnvelope,
+    security_descriptor: bytes,
+    sid: str,
+) -> bytes:
+    # Generate cek and encrypt our payload.
+    cek = AESGCM.generate_key(bit_length=256)
```

Comment:
It would be nice to keep the crypto algorithms inside `_crypto.py`. I'm also curious is there any benefit for doing this with `AESGCM.generate_key` over `os.urandom`?

Reply:
I followed the example in the [documentation](https://cryptography.io/en/latest/hazmat/primitives/aead/#cryptography.hazmat.primitives.ciphers.aead.AESGCM): `AESGCM.generate_key` for the key and `os.urandom()` for the IV.